tests/conftest.py

```diff
@@ -81,7 +81,24 @@ def java_command_line(tmp_path: Path, java_args: List[str]) -> List[str]:
 
 
 @fixture
-def command_line(runtime: str, java_command_line: List[str]) -> List[str]:
+def dotnet_command_line(tmp_path: Path) -> List[str]:
+    class_path = tmp_path / "dotnet"
+    class_path.mkdir()
+    class_path = class_path / "Fibonacci"
+    class_path.mkdir()
+    chmod_path_parts(class_path, stat.S_IRGRP | stat.S_IROTH | stat.S_IXGRP | stat.S_IXOTH)
+    subprocess.run(["cp", str(CONTAINERS_DIRECTORY / "dotnet/Fibonacci.cs"), class_path])
+    subprocess.run(["dotnet", "new", "console", "--force"], cwd=class_path)
+    subprocess.run(["rm", "Program.cs"], cwd=class_path)
+    subprocess.run(["dotnet", "build", "-c", "Release"], cwd=class_path)
+    # dotnet building server (Roslyn) is waiting for its parent process to comple, which is pytest, it must be killed
+    # not to interfere with the actual test
+    subprocess.run(["pkill","-f","Roslyn"])
```

Comment:
Preferably we will avoid `pkill -f` on the user machine - if they have Roslyn running, for example. Or anothing that matches it.
Why does it interfere with the test? Because Roslyn is also also profiled by dotnet-trace? Does it affect the test results?

Reply:
By design, these project generators or code compilators seem to wait for the parent process to end (in our case - pytest), and then the test requires to have one matching dotnet process, so  AssertionError occurs. If test reqiures no other dotnet processes to pass, I thought that kind of solution will be viable. I'll try to think of something else.